styles/status.cafe/catppuccin.user.css

```
@@ -0,0 +1,194 @@
+/* ==UserStyle==
+@name Status.cafe Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/status.cafe
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/status.cafe
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/status.cafe/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Astatus.cafe
+@description Soothing pastel theme for status.cafe
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+
+@var checkbox colors "Accent color links" 1
+@var checkbox break "Break long statuses" 1
+==/UserStyle== */
+
+@-moz-document url("https://status.cafe/"), url("https://status.cafe/settings"), url("https://status.cafe/manage"), url("https://status.cafe/login") {
```

```suggestion
@-moz-document domain("status.cafe") {
```
Is there an issue doing just this?

---

Yes, since user profiles (`status.cafe/users/<username>`) have custom css. Although, I swapped that with a more efficient regexp statement that only excludes that subdomain.